bot.js

```diff
@@ -0,0 +1,25 @@
+//Login to Discord
+const Discord = require('discord.js');
+const bot = new Discord.Client();
+const config = require('./config.json');
+bot.login(config.auth_token);
```

Comment:
Add a new line for readability

```suggestion

bot.login(config.auth_token);
```